llvm/utils/lit/tests/Inputs/shtest-env-positive/env-no-subcommand.txt

```diff
@@ -6,32 +6,32 @@
 # NO-ARGS: BAR=2
 # NO-ARGS: FOO=1
 # NO-ARGS: QUX=3
-
+   
```

Comment:
Nit, you've added trailing whitespace again here and on many other lines in this file. Please fix!

Reply:
Fixed.